docs/developer-docs/docs/build-a-keychain/operate-a-keychain/create-a-keychain.md

```
@@ -98,12 +98,12 @@ You can skip this guide and test a preconfigured Keychain. Just run a local node
    ```
    :::
 
-5. Fund your key using [Buenavista faucet](https://faucet.buenavista.wardenprotocol.org) and the public address returned in the previous step.
+5. Fund your key using [Chiado faucet](https://faucet.chiado.wardenprotocol.org) and the public address returned in the previous step.
 
-5. Check your balance. Here and in other commands, you need to add the `--node` flag with an RPC URL for connecting to Buenavista. 
+5. Check your balance. Here and in other commands, you need to add the `--node` flag with an RPC URL for connecting to Chiado. 
    
    ```
```

Markdownlint

Fenced code blocks should have a language specified

(MD040, fenced-code-language)